iceberg/iceberg-handler/src/test/queries/positive/iceberg_add_complex_column.q

```diff
@@ -0,0 +1,15 @@
+CREATE TABLE t_complex (id INT) STORED BY ICEBERG;
+
+INSERT INTO t_complex (id) VALUES (1);
+
+ALTER TABLE t_complex ADD COLUMNS (col1 STRUCT<x:INT, y:INT>);
+
+SELECT * FROM t_complex ORDER BY id;
```

Comment:
i think would be nice if we add insert with some dummy struct after the schema change. same applies to map and array. 
And then execute just single select on a table.